src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double area();

```

Comment:
названия методов должны начинаться с глаголов
```suggestion
    double getArea();

```